tools/generate-templates.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -euo pipefail
+
+rm -rf templates || true
+mkdir -p templates
```

Comment:
If we create some temporary files, it's better to add them in `.gitignore` as well.

As we may run `tox -e lint` locally, it would generates this temporary files.

Reply:
We can't put it into .gitignore because pre-commit will ignore it in linting